### DISCUSSION TITLE
Skip automatically if all percentages below given number?

### DISCUSSION THREAD

**Discussion by Hexadigital** [beetbox/beets#4204] [Q&A] [answered]
Hello! I apologize if there is something about this in the documentation and I just can't find it, but I was wondering if it is possible to have beets automatically skip if no good match is found (like the quiet flag does) without skipping if the match percentage is high enough to be worth looking at.

Something like this:
96%+ = automatically apply
80-95.9% = prompt for user action
0-79.9% = skip

**Comment by sampsyo** [comment, marked as answer]
Yes! Try setting `import.none_rec_action` to `skip`. https://beets.readthedocs.io/en/stable/reference/config.html#none-rec-action

**Comment by Hexadigital** [comment]
Thank you very much! That works perfectly.
I saw that option but thought it was for when no matches were found at all (like when you try importing something without any tags).

**Reply by sampsyo** [reply]
It is certainly confusingly named! 😆
